src/app/modules/applications/components/servable-names/servable-names.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { ApplicationsFacade } from '@app/core/facades/applications.facade';
+import { ServablesFacade } from '@app/core/facades/servables.facade';
+import {
+  Application,
+  Servable,
+  ModelVersion,
+} from '@app/core/data/types';
+
+import { Observable } from 'rxjs';
+
+@Component({
+  selector: 'hs-servable-names',
+  templateUrl: './servable-names.component.html',
+  styleUrls: ['./servable-names.component.scss']
+})
+export class ServableNamesComponent implements OnInit {
+  application$: Observable<Application>;
+  servables$: Observable<Servable[]>;
+
+  constructor(
+    private readonly applicationFacade: ApplicationsFacade,
+    private readonly servableFacade: ServablesFacade
+  ) { }
+
+  ngOnInit() {
+    this.application$ = this.applicationFacade.selectedApplication();
+    this.servables$ = this.servableFacade.allServables();
+  }
+
+  get servableNames(): string[] {
```

Comment:
Move that logic to parent component (smart component aka container), and communicate with that component via @Input
i.e you can create @Input servableNames: string[] = []